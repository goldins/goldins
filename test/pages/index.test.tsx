import React from 'react';
import { render } from '../testUtils';
import Index from '../../pages';

describe('Index page', () => {
  it('clicking button triggers alert', () => {
    const { asFragment } = render(<Index />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
