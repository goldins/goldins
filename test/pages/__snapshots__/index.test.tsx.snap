// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page clicking button triggers alert 1`] = `
<DocumentFragment>
  <div
    class="css-12mft5n"
  >
    <div
      class="css-12mft5n"
    >
      <main>
        <h1
          class="css-1y1lo0n"
        >
          Simon.Goldin
        </h1>
      </main>
      <footer
        class="css-8u3q9h"
      >
        Footer
      </footer>
      <a
        href="./blog"
      >
        Blog
      </a>
      <a
        href="./projects"
      >
        Projects
      </a>
    </div>
  </div>
</DocumentFragment>
`;
